### PR TITLE
[ESPnet2]  remove model_size arguments from noam lr scheduler

### PR DESCRIPTION
**Pull request by kamo-naoyuki**
Originally "model_size" is derived from the model, when instantiation of this class, but this parameter is just argument and it's not related to actual model size in the current implementation in ESPnet2.

https://github.com/espnet/espnet/blob/e88a477cb72be7e5a03595ead5c233f8d211f6b6/espnet/asr/pytorch_backend/asr.py#L431-L433

Indeed, this model_size is just multiplied to learning rate and this parameter seems redundant. I removed this parameter, so the learning rate must be multiplied `model_size ** 0.5` from the previous configuration.

- Before PR, base lr is:
 `lr / model_size ** 0.5`

- After PR, Base lr is:
   `lr`

Where lr is the optimizer's lr.

### CODE REVIEW

File: espnet2/schedulers/noam_lr.py [hunk 1]
```diff
@@ -39,8 +35,6 @@ def __init__(
     def get_lr(self):
         step_num = self.last_epoch + 1
         return [
-            lr
-            * self.model_size ** -0.5
```
Comment (sw005320) [inline espnet2/schedulers/noam_lr.py]: Does this mean that `lr` would be changed from the previous one by a factor of `self.model_size ** -0.5`?
Does this modification change `lr` tuning range?

Comment (kamo-naoyuki) [reply espnet2/schedulers/noam_lr.py]: Sorry, this implementation had a bug, that is not my original intention of PR. The lr will be multiplied by `self.model_size ** -0.5` at every steps.

**Comment by kamo-naoyuki** [comment]
I'll reopen.
